src/actions/index.js

```diff
@@ -417,8 +417,21 @@ function updateOutput(
             dependency.output.property === outputProp
         )
     );
+    const validKeys = Object.keys(paths);
```

Comment:
By convention, we use [`ramda`](https://ramdajs.com/docs/) as our "standard library" for all things JavaScript, even when a JavaScript method exists. So here, this would be `R.keys(paths)`